### DISCUSSION TITLE
Wish List: Add custom social icon

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#20] [General]
I don't see any way to do this now, but I would love the ability to specify a custom icon for social media links.
![image](https://github.com/buttondown-email/discussions/assets/12089920/65e0844d-1dab-4c9f-ab40-75cb9df3c7d0)

**Comment by jmduke** [comment]
Agreed; closing in favor of https://github.com/buttondown-email/roadmap/issues/2423.
